src/commands.py

```diff
@@ -267,13 +267,15 @@ async def bin_from_ascii(self, ctx):
         if len(args) < 2:
             await ctx.send("Please provide all necessary arguments. !fromascii <ASCII string>")
             return
-        ascii_string = " ".join(args[1:])
-        new_bin = ''.join(format(i, 'b') for i in bytearray(ascii_string, encoding='utf-8'))
+        ascii_string = " ".join(args[1:])  # remove command part
+        # Use bytearray() to convert to binary value
+        # Then use format() to pad the front with 0s to make up to 7
+        bin_string = "".join("{:0>7b}".format(i) for i in bytearray(ascii_string, encoding='utf-8'))
```

Comment:
It's really just this bit. The rest are some stylistic changes

Reply:
```py
"{:0>7b}".format(i)
```  
`:` -> To denote the use of *format_spec* (similary, `!` would denote the use of a `conversion` field)  
`0` -> Pad out with 0s  
`>` -> Right-justify  
`7` -> Make up to 7  
`b` -> Binary format

See [Format Specification Mini-Language](https://docs.python.org/3/library/string.html#format-specification-mini-language) on the Python official docs for more details.